settings/storage.py

```diff
@@ -0,0 +1,83 @@
+import json
+import sqlite3
+from datetime import datetime
+
+from settings.detection import DetectionSettings
+from utils.logging import main_logger
+
+logger = main_logger.getChild("settings.storage")
+
+
+class SettingsStorage:
```

Comment:
Why do we open and close the db each time instead of keeping it open?

Reply:
also docs

Reply:
Will add a TODO